VeganVirus/VeganVirus/Animation.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+#include "Draw.h"
+#include <vector>
+
+class Animation
+{
+public:
+	// Animation Constructor
```

Comment:
I would recommend using `/**/` instead of `//` .but if you found your way more fun you can keep it :)

Reply:
I prefer using `//` 😎